Frontend/src/components/TeacherPage.jsx

```
@@ -76,20 +85,18 @@ export default function TeacherDashboard() {
-    {
-      title: t('feePayment'),
-      description: t('deadlineExtendedTo30thOct')
-    }
-  ]);
+  const [announcements, setAnnouncements] = useState([]);
+  const [recentAnnouncements, setRecentAnnouncements] = useState([]);
+  const [oldAnnouncements, setOldAnnouncements] = useState([]);
+  const [showPreviousAnnouncements, setShowPreviousAnnouncements] = useState(false);
+  const [newAnnouncementTitle, setNewAnnouncementTitle] = useState("");
+  const [newAnnouncementDescription, setNewAnnouncementDescription] = useState("");
+  const [announcements, setAnnouncements] = useState([]);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'announcements'. Consider to delete it or rename it.

'announcements' is defined here:


(lint/suspicious/noRedeclare)